src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -356,6 +356,8 @@ public static function getSignature(array $a)
                     $signature .= 10 > \strlen($v) && !str_contains($v, '\\') ? "'{$v}'" : "'…".\strlen($v)."'";
                 } elseif (\is_bool($v)) {
                     $signature .= $v ? 'true' : 'false';
+                } elseif (\is_object($v)) {
```

Comment:
What if the object is an Enum instance ?

Reply:
I just gave it a try: an Enum is seen as a const, aka `$param->isDefaultValueConstant()` returns true